app/assets/stylesheets/simple_pages.scss

```diff
@@ -1,53 +1,27 @@
-h1, h2, p {font-family: monospace}
-
-h1 {color: darkred}
-
-h1, h2, p {text-align: center}
-
-footer {text-align: right}
-
-nav ul li {
-    display: inline-block;
-    float: right;
-}
-
-nav {
-    background-color: black;
-    position: fixed;
-    width: 100%;
-    top: 0;
-    padding: 20px 10px 20px;
-}
-
-nav ul li a {
-    color: white;
-    text-transform: uppercase;
-    letter-spacing: 0.05em;
-    padding: 20px 10px 20px;
-}
-
-body {
-    padding-top: 50px;
+.footer {
+  padding-top: 15px;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 60px;
+  background-color: #f5f5f5;
+  .copyright {
+    text-align: right;
+  }
+}
+
+.wrapper {
+  /* Margin bottom by footer height */
+  position: relative;
+  margin: 0 auto;
+  padding-bottom: 6rem;
+  height: 100%;
+  min-height: 100%;
 }
 
-* {
-  margin: 0;
-}
 html, body {
+  position: relative;
+  margin: 0 auto;
   height: 100%;
+  min-height: 100%;
 }
```

Comment:
The site footer has been giving me quite some grief. This can be revisited, but I didn't want to spend a whole lot of time on this.